app/web/features/auth/locales/en.json

```
@@ -105,7 +105,8 @@
     }
   },
   "feedback_form": {
-    "header": "Have your say"
+    "header": "Have your say",
+    "skiptitle":"Not interested? Click <a>here</a> to skip this step."
```

```suggestion
    "skip_title":"Not interested? Click <1>here</1> to skip this step."
```
Please space separate words like in the surrounding examples.

Also, you might have to do something like this rather than putting an `<a>` here. The `<1>`s allow the bits to be replaced with the correct element in the code.

---

got it